client/client.go

```diff
@@ -454,3 +454,18 @@ func (client *Client) CreateUsers(options []*CreateUserOptions) ([]*CreateUserRe
 	}
 	return results, nil
 }
+
+type DeviceUser struct {
+	Username string `json:"username"`
+	Email    string `json:"email"`
+}
+
+// Users returns the local users.
+func (client *Client) Users() ([]DeviceUser, error) {
```

Comment:
We should probably use `[]*User` here (pointer), so we encourage manipulating by identity.